### PR TITLE
Fixed feature change ajax

### PR DESCRIPTION
**Pull request by aleeks**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Fixed feature change ajax |
| Type? | bug fix |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1636 |
| How to test? | Go to product, change feature (think to change a feature wich have default value) |

#### Important guidelines
- Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
- Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
- Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 1]
```diff
@@ -1091,7 +1091,6 @@
 
 {% block stylesheets %}
   {{ parent() }}
-  <link rel="stylesheet" href="{{ asset('themes/default/css/bundle/module/module.css') }}" />
```
Comment (aleeks) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: This file doesn't exist ^^

Comment (antoin-m) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: If you remove everything in a `{% block %}` but `parent()` you can remove the whole block

**Review by antoin-m — APPROVED** [review]

**Comment by antoin-m** [comment]
Thank you :)

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Pull request merged**